### PR TITLE
Quick fix malware detection Bug

### PR DESCRIPTION
**Pull request by superpoussin22**
### Describe the Pull Request

```
Preivous method produce False-Positive. 
This new version normalize also url in apk to extract the domain the right way 

```

### Checklist for PR

- [X] Run MobSF unit tests and lint `tox -e lint,test`.
- [X] Tested Working on Linux, Mac, Windows, and Docker
- [X] Make sure build is passing on your PR. [Build Status](https://travis-ci.com/MobSF/Mobile-Security-Framework-MobSF/pull_requests)

### CODE REVIEW

File: MalwareAnalyzer/views/domain_check.py [hunk 1]
```diff
@@ -91,6 +91,24 @@ def malware_check(urllist):
...
+    try:
+        domain = ''
+        parsed_uri = urlparse(url)
+        if not urlparse(url).scheme:
+            url = '//' + url
+            parsed_uri = urlparse(url)
+            domain = '{uri.netloc}'.format(uri=parsed_uri)
+        if (len(domain) > 2
+                and '.' in domain
+                and (domain.endswith('.') is False
```
Comment (ajinabraham) [inline MalwareAnalyzer/views/domain_check.py]: @superpoussin22 what is this check for?

Comment (superpoussin22) [reply MalwareAnalyzer/views/domain_check.py]: will try: on one side we have a list of url and in the other only one url

Comment (superpoussin22) [reply MalwareAnalyzer/views/domain_check.py]: @ajin: Rework down let me know what you think

File: MalwareAnalyzer/views/domain_check.py [hunk 2]
```diff
@@ -91,6 +91,34 @@ def malware_check(urllist):
 # Helper Functions
 
 
+def verify_domain(checkeddom):
+    try:
+        if (len(checkeddom) > 2
+                and '.' in checkeddom
+                and (checkeddom.endswith('.') is False
+                     and re.search('[a-zA-Z0-9]', checkeddom))):
+            return(True)
```
Comment (ajinabraham) [inline MalwareAnalyzer/views/domain_check.py]: @superpoussin22  remove unnecessary `(` and `)`

Comment (ajinabraham) [inline MalwareAnalyzer/views/domain_check.py]: @superpoussin22  remove unnecessary `(` and `)`

File: MalwareAnalyzer/views/domain_check.py [hunk 3]
```diff
@@ -91,6 +91,34 @@ def malware_check(urllist):
...
+        else:
+            return(False)
```
Comment (ajinabraham) [inline MalwareAnalyzer/views/domain_check.py]: @superpoussin22  remove unnecessary `(` and `)`

File: MalwareAnalyzer/views/domain_check.py [hunk 4]
```diff
@@ -91,6 +91,34 @@ def malware_check(urllist):
...
+    """Get Domain."""
+    try:
+        domain = ''
+        parsed_uri = urlparse(url)
+        if not urlparse(url).scheme:
+            url = '//' + url
+            parsed_uri = urlparse(url)
+            domain = '{uri.netloc}'.format(uri=parsed_uri)
+        if verify_domain(domain):
+            return(domain)
```
Comment (ajinabraham) [inline MalwareAnalyzer/views/domain_check.py]: @superpoussin22  remove unnecessary `(` and `)`

File: MalwareAnalyzer/views/domain_check.py [hunk 5]
```diff
@@ -91,6 +91,34 @@ def malware_check(urllist):
...
+    except Exception:
+        logger.exception('[ERROR] Extracting Domain form URL in APK')
```
Comment (ajinabraham) [inline MalwareAnalyzer/views/domain_check.py]: Not just APK, but source as well so. Reword to 

`[ERROR] Extracting domain form URL`

**Comment by superpoussin22** [comment]
@ajinabraham changes done

**Pull request merged**